docs/Concepts/Merge.md

```diff
@@ -36,26 +39,27 @@ the consensus layer.
 Consensus clients serve [REST API](https://docs.teku.consensys.net/en/stable/Reference/Rest_API/Rest/) requests and
 communicate with each other in a peer-to-peer network.
 
-### Run a node
+## What happens during The Merge
+
+Before The Merge, the execution and consensus clients' configurations will be
+[updated](../HowTo/Upgrade/Prepare-for-The-Merge.md#update-besu) to listen for a certain total terminal difficulty (TTD)
```

Comment:
Perhaps link to: https://ethereum.org/en/glossary/#terminal-total-difficulty
Or preferably, add the description in a note so users don't have to click away from the page